packages.cmake

```diff
@@ -135,6 +68,13 @@ macro(build_cpack_config)
     create_packagecloud_publish_target(${_pc_user} ${_pc_repo} ${_pc_distro} ${_package_output})
   endforeach()
 
+  # Add a target for uploading the package to Amazon S3
+  set(_s3_channel "ci-builds")
+  set(_version "${OTC_VERSION}-${BUILD_NUMBER}")
```

Comment:
should this be ```OTC_BUILD_NUMBER```? 🤔

Reply:
No, although I can see how that is confusing. The CMake code reads `ENV{OTC_BUILD_NUMBER}` and stores it in a CMake variable called `BUILD_NUMBER`: https://github.com/SumoLogic/sumologic-otel-collector-packaging/blob/upload-s3/version.cmake#L18.